sql/sqltool/hidden_windows.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+package sqltool
```

Comment:
No build flag for windows?

Reply:
The name itself is enough. `hidden_windows.go` will only be compiled if `GOOS == windows`.